lib/diagnostic.js

```diff
@@ -14,12 +14,22 @@
 'use strict';
 
 const fs = require('fs'); // jshint ignore: line
+const inFile = process.argv[2];
 
-const sumLines = (filename, callback) => {
-  /* your response here */
-  callback();
+const sumLines = ('integers.txt', callback) => {
+  fs.readFile('/data/integers.txt', 'utf8', (error, data));
```

Comment:
Recall that the 3rd argument of readFile is a FUNCTION, whose 2 arguments are `error` and `data`. That's not what you wrote here.